pages/index.js

```diff
@@ -1,15 +1,55 @@
-import { fetchPost, getReadingTime, formateDate } from "../lib/post";
+import {
+  fetchPost,
+  getReadingTime,
+  formateDate,
+  searchPost,
+} from "../lib/post";
 import Image from "next/image";
 import Link from "next/link";
 import MarkdownView from "react-showdown";
 import Loader from "./../component/loader";
 import { Dialog, Transition } from "@headlessui/react";
-import { Fragment, useState } from "react";
-import config from "../config";
+import { useCallback, useRef, Fragment, useState } from "react";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faMessage } from "@fortawesome/free-solid-svg-icons";
 
 export default function Home({ posts, status }) {
+  //Search Input
+
+  const searchRef = useRef(null);
+  const [query, setQuery] = useState("");
+  const [active, setActive] = useState(false);
+  const [results, setResults] = useState([]);
```

Comment:
searchBlog is more suitable instead of results

Reply:
Can be `searchedBlogs`